Bank-Assesment/.vs/KolliSai_CSharp101/v17/DocumentLayout.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please add .vs directory in .gitignore file

Reply:
Added Can you check once.